src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -193,12 +193,37 @@ public static void onPlayerDestroyItem(Player player, @NotNull ItemStack stack,
      * Vanilla calls to {@link Mob#finalizeSpawn} are replaced with calls to this method via coremod.<br>
      * Mods should call this method in place of calling {@link Mob#finalizeSpawn}. Super calls (from within overrides) should not be wrapped.
      * <p>
-     * Returns the SpawnGroupData from this event, or null if it was canceled.
+     * When interfacing with this event, write all code as normal, and replace {@link Mob#finalizeSpawn} with {@link #onFinalizeSpawn}.<p>
+     * As an example, the following code block:
+     * 
+     * <blockquote><pre>
+     * var zombie = new Zombie(level);
+     * zombie.finalizeSpawn(level, difficulty, spawnType, spawnData, spawnTag);
+     * level.tryAddFreshEntityWithPassengers(zombie);
+     * if (zombie.isAddedToWorld()) {
+     *     // Do stuff with your new zombie
+     * }
+     * </blockquote></pre>
```

Comment:
Surround code blocks with `{@code}`. Additionally, the ordering of your tags are incorrect (the closing tag for `<pre>` should be before the closing tag for `<blockquote>`), and I don't think the blockquote tag is needed (but you may choose to leave it in).

~~One day we'll have the `{@snippet}` tag...~~

```suggestion
     * <pre>{@code
     * var zombie = new Zombie(level);
     * zombie.finalizeSpawn(level, difficulty, spawnType, spawnData, spawnTag);
     * level.tryAddFreshEntityWithPassengers(zombie);
     * if (zombie.isAddedToWorld()) {
     *     // Do stuff with your new zombie
     * }
     * }</pre>
```